setup.py

```diff
@@ -172,16 +169,18 @@ def run(self):
         sdist.run(self)
 
 
+install_requires = ["numpy", "pillow"]
+
 extras_require = {
+    "dev": ["invoke", "black", "flake8", "pytest", "pytest-cov", "psutil"],
+    "docs": ["sphinx", "numpydoc"],
+    "ffmpeg": ["imageio-ffmpeg"],
```

Comment:
I really like this reorganization. A thought: afaik there is only `imageio-ffmpeg` as an additional dependency for more formats. In the future, there could be more. Could/would it make sense to add a key called `full` (naming suggestions welcome) so that we can install all formats via `pip install imageio[full]`? Whereas `pip install imageio` would only install commonly used ones (I guess pillow?)

Reply:
There already is an `imageio[full]`, defined slightly further down :)

Reply:
Ah! I missed that one. My bad.